### DISCUSSION TITLE
Ability to save custom search ranges

### DISCUSSION THREAD

**Discussion by FalcusDev** [RetroAchievements/RAIntegration#906] [Ideas]
Within the memory inspector you can limit the search range or leave it on all/system memory.
The ability to restrict the search to a range of where you know data is held is extremely useful at times, so being able to save these searches so that you can access them in a similar manner to all/system memory would be useful on a game by game basis where you know certain data is held within a specific range.

**Comment by Jamiras** [comment]
implemented in #1256.
